### DISCUSSION TITLE
Why ReadFile() too slow?

### DISCUSSION THREAD

**Discussion by Khasehemwy** [assimp/assimp#4661] [Q&A] [answered]
main code:
```cpp
Assimp::Importer import;  
const aiScene* scene = import.ReadFile(path, aiProcess_Triangulate | aiProcess_FlipUVs);
```

using lib : **assimp-vc143-mtd.lib**
using obj : https://learnopengl-cn.github.io/data/nanosuit.rar
using IDE : Visual studio 2022
using IDE solution configuration : Debug/x64
OS: Windows10 - 21H1

I had to wait 3 minutes to load that model...

(As you might guess this problem caused by Debug mode, but I also try Release mode, problem remains)

**Comment by kimkulling** [comment, marked as answer]
Try the latest master. The obj-loader had a glitch, which had caused high load issues. And this had increased the import time dramatically.

But it is fixed right now.
